### DISCUSSION TITLE
Upgrading Bootstrap library from 1.4 to the latest

### DISCUSSION THREAD

**Discussion by Legend4an** [twbs/bootstrap#32072] [Q&A] [answered]
Hello folks,

I want to upgrade my Bootstrap library from 1.4 to the latest. My question is that what is the best way for doing this? should I go through the changelogs for each upgrade? Would it be a lot of work?

Also, is it possible that if I only upgrade the javascript library and keep the older version of CSS?

**Comment by mdo** [comment, marked as answer]
Oh wow, that's a helluva update. There's no super smooth path to upgrading from being more than three major releases behind. It's gonna be slow and manual.

> 

Our JS is wildly different in v4.x and v5.x than v1.x. Not possible to do that.
